drivers/hmis/app/graphql/mutations/create_form_definition.rb

```diff
@@ -34,6 +30,8 @@ def resolve(input:)
         **attrs,
       )
 
+      access_denied! unless policy_for(definition, policy_type: :form_definition).can_create?
```

Comment:
For discussion:
1. pass initialized record to policy, or
2. support nil resource (` policy_for(nil, policy_type: :form_definition).can_create?`), or
3. something else?

Maybe it depends on the case. In this case their isn't an easy "parent" policy to ask